processes/executive-copy-creation-instructions.md

```diff
@@ -0,0 +1,128 @@
+# Executive Copy Creation Instructions
+
+## Prepare GitHub
+
+1. Sync Community Git.
+   
+3. Ensure you are working from the latest master branch.
```

Comment:
Skips 2 and is probably not necessary.

Reply:
Not sure why the numbers moved. There was nothing between 1 and 3 :)